src/Typeahead.react.js

```diff
@@ -137,6 +137,11 @@ const Typeahead = React.createClass({
      * to control the component via its parent.
      */
     selected: PropTypes.array,
+
+    /**
+     * Propagate <Enter> to parent form
+     */
+    formSubmit: PropTypes.bool,
```

Comment:
👍 , `submitOnEnter` or `submitFormOnEnter` is perfectly well, actually my first name was: `supportFormSubmit`, but I have cut it to shorter form